ReduxTestTask/Classes/NewsList/ViewModel/Middleware/NewsListFetchMiddleware.swift

```diff
@@ -93,6 +93,7 @@ extension NewsList {
             Observable
                 .just(article)
                 .map ({ article -> Action in
+                    showArticle.onNext(article)
```

Comment:
map should be used for transformations, not for side effects. Same for flatMap.

Use do(onNext:) for side effects